lib/public/Share/IShare.php

```diff
@@ -385,11 +385,11 @@ public function getNote();
 	/**
 	 * Set the expiration date
 	 *
-	 * @param null|\DateTime $expireDate
+	 * @param \DateTime|string|null $expireDate
```

Comment:
```suggestion
	 * @param \DateTime|''|null $expireDate
```
To avoid any other string than the empty string. I'm questioning a bit if this couldn't be solved differently because two "null" values are not great

Reply:
Which is why i would have added another prop for this. Also helps to improve the enforcement handling above